lib/nylas/resources/calendars.rb

```diff
@@ -52,7 +53,8 @@ def create(identifier:, request_body:)
     # @param identifier [String] Grant ID or email account in which to update an object.
     # @param calendar_id [String] The id of the calendar to update.
     #   Use "primary" to refer to the primary calendar associated with grant.
-    # @param request_body [Hash] The values to update the calendar with
+    # @param request_body [Hash] The values to update the calendar with.
+    #   This can include a `notetaker` object with settings for the Notetaker bot.
```

Comment:
This is being nit-picky but should we mention that these also contain rules for the notetaker calendar sync

Reply:
Updated!